internal_ws/ykbh/src/lib.rs

```diff
@@ -9,6 +11,8 @@ use yktrace::sir::SIR;
 /// Stores information needed to recreate stack frames in the SIRInterpreter.
 pub struct FrameInfo {
     /// The symbol name of this frame.
+    /// i might have misunderstood this, but does every function call in the stopgap interpreter
+    /// require mallocing a String?
```

Comment:
Lukas and I talked about whether it was possible to borrow a slice directly from the SIR. It might be possible, as (currently) we never unload a SIR body once it's loaded. If we ever want to evict SIR, we must copy.

Reply:
@ptersilie why don't we just store the corresponding `Arc` in this struct?

Reply:
Haven't tried that. When I wrote this I had other things to worry about. But there's definitely room for optimisations in the blackholing code.

Reply:
Fixed in 0163c4d.